### DISCUSSION TITLE
Help:how to show document metadata in citations of LLM answers

### DISCUSSION THREAD

**Discussion by soulzzz** [langgenius/dify#8511] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

deleted-

**Comment by soulzzz** [comment]
closed due to resolved

**Reply by yangxuan21** [reply]
have any relevant reference documents？

**Comment by simongaocd** [comment]
Hi，how to show metadata in citations?
